### ISSUE TITLE
Missing heading for phases of consultation

### ISSUE THREAD

**Issue by gillianlevine** [WorldHealthOrganization/smart-emcare#28]
There is no clear heading displayed indicating what phase of the consultation the user is in. This is important because a health worker needs to know if something should be asked of the caregiver (eg. "symptoms" or evaluated during a physical exam "signs". Each phase of the consultation needs a clear heading for the user to see what phase they are in and to describe that phase as per IMCI language when applicable. For example, Classifications are displayed without any heading that says classification, so it looks to users like another set of questions for them to review and respond to, rather than the classification

**Comment by delcroip** [comment]
there is the questionnaire name on top for now to help with troubleshooting

**Comment by ppanchal-argus** [comment]
Thanks for this! Replaced questionnaire name with appropriate heading in header. Will be released with v1.0.10

**Comment by kaledip** [comment]
In the headers codes are also seen for example emcare.B23.Classification . Can we name it as only "EmCare.Classifications" or "Classifications".
This applied to all questionnaires

**Comment by ppanchal-argus** [comment]
@kaledip Yes I figured it was a bug. Fixed it and releasing a dev apk for this.

**Comment by kaledip** [comment]
@ppanchal-argus Headings appear ok during consultation, but during registration its shows emcare.e.registration.

![Image](https://user-images.githubusercontent.com/122801361/229719666-0eed5a0a-7855-49d9-b6a9-9d4e1f0c4ab8.jpg)

**Comment by kaledip** [comment]
@ppanchal-argus  the heading during the patient registration still says- emcareb.registration.e
Screenshot as attached.

**Comment by kaledip** [comment]
@ppanchal-argus  is the next dev version available on the same link?

**Comment by kaledip** [comment]
Retested on dev 1.1.9. Look ok
